exo-go/src/docker_compose/index.go

```diff
@@ -0,0 +1,43 @@
+package dockerCompose
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os/exec"
+)
+
+// RunAllImages runs all the docker images
+func RunAllImages(env []string, cwd string, write func(string)) (*exec.Cmd, *bytes.Buffer, error) {
```

Comment:
I saw that but I don't know if there is away to get the command output (for the logger) if we use this. The repo doesn't have very useful examples but I think we'd do something like [this](https://github.com/docker/libcompose/blob/master/example/main.go#L15). Based on the [struct of `options.Up`](https://godoc.org/github.com/docker/libcompose/project/options#Up), I don't know if there is a way to pass in environment variables. Also, we are using `docker-compose` version 3 and it looks like that they don't have support for that yet (see this [issue](https://github.com/docker/libcompose/issues/474))